### PR TITLE
feat(server): adding a /ver.json route

### LINKED ISSUE #214: Add ver.json route to dev/stage/prod servers
Not sure where this is generally injected, but we'll need this set up so we know what'd deployed in each environment.

ie: https://chronicle.dev.mozaws.net/ver.json

Here's what we do for [fxa-content-server](https://latest.dev.lcip.org/ver.json):
```js
{
  version: "0.29.0",
  commit: "71aca8403cbfc717572303f793f131180a317617",
  l10n: "bb53fd5e8f",
  tosPp: "29ec46747a"
}
```

Where `version` is from the package.json file, and `commit` is the currently deployed Git SHA.
The `l10n` and `tosPp` are Git SHAs to the respective bower repos.

### PR DESCRIPTION
**Pull request by pdehaan**
Fixes #214

### CODE REVIEW

**Comment by pdehaan** [comment]
```sh
Running "watch" task
Waiting...
chronicle.server.ver.json.INFO: version set to: 0.1.0
chronicle.server.ver.json.INFO: commit hash set to: c662ad303039faca5ee535a531b962521eaef3dc
```

**Comment by pdehaan** [comment]
Magic lines in Travis: https://travis-ci.org/mozilla/chronicle/builds/50402323#L944-L945
```
944: chronicle.server.ver.json.INFO: version set to: 0.1.0
945: chronicle.server.ver.json.INFO: commit hash set to: 98e0742dbe6313fa62c709cda8cbfd9c3148a80c
```

**Comment by pdehaan** [comment]
r? @6a68 @nchapman 

"Works for me (and Travis)"
May or may not need a bit of tweaking for the dev/stage server.

**Comment by 6a68** [comment]
This looks awesome! Do you think it's generalizable, something we could split into a separate npm module? It seems like everything but the routing/controller bit could be reused across a bunch of fxa projects, whether they're using express or hapi. Not necessarily something to do right this second or in this PR, but might be cool to use npm instead of copypasta for these common tools :-)

**Comment by 6a68** [comment]
(I'll take a closer look tomorrow)

**Comment by pdehaan** [comment]
@6a68 It was largely a cut-n-paste job with a bit of tweaking and deleting.
Not entirely sure how generalizable it would be since it has a lot of file paths, some mozlog usage, and may be very specific to the app as to what hashes you want to put in that ver.json file (for example, fxa-content-server also displays the bower SHA info for the l10n files and legal files -- which I'll assume we may need to add back in the future once we get the l10n stuff set up).

File: server/controllers/ver.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  return promise;
+}
+
+// seed the info on startup.
+getVersionInfo();
+
+var verJsonController = {
+  get: function (request, reply) {
+      getVersionInfo().then(function (versionInfo) {
+        // charset must be set on json responses.
```
Comment (6a68) [inline server/controllers/ver.js]: not sure about this comment

File: server/controllers/ver.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
```
Comment (6a68) [inline server/controllers/ver.js]: probably good to add a comment indicating that it's stolen from another repo ;-)

File: server/controllers/ver.js (hunk 1 above)
Comment (6a68) [inline server/controllers/ver.js]: ...but it's fine

File: server/controllers/ver.js (hunk 2 above)
Comment (6a68) [inline server/controllers/ver.js]: oh wait, just saw your [other comment](https://github.com/mozilla/chronicle/pull/325#issuecomment-74117037) about this, it's fine

**Comment by 6a68** [comment]
@pdehaan nice work! love that pasta

**Comment by 6a68** [comment]
works for me, merging!

**Pull request merged**